common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilterImpl.java

```diff
@@ -212,12 +227,10 @@ public BloomFilter mergeInPlace(BloomFilter other) throws IncompatibleMergeExcep
 
     if (this.numHashFunctions != that.numHashFunctions) {
       throw new IncompatibleMergeException(
-        "Cannot merge bloom filters with different number of hash functions"
+              "Cannot merge bloom filters with different number of hash functions"
```

Comment:
nit: seem to be unnecessary changed indentation.

Reply:
reverted. thanks!